releasenotes/notes/prepare-0.45-50118e5e6d110302.yaml

```
@@ -0,0 +1,41 @@
+---
+prelude: |
+  Qiskit 0.45.0 is the last feature release before 1.0. It prepares the ground for the API
+  changes we are planning for our first major release, including many removals of
```

I'm not sure if this is better, but my first thought reading this was "all our releases are major" :).

```suggestion
  changes we are planning for our first major version release, including many removals of
```